plugins/jobs/web_client/views/JobGraphWidget.js

```diff
@@ -67,25 +68,29 @@ const JobGraphWidget = View.extend({
             let withForEachJob = width / numberOfJobs;
             // if the width for each job is less than 20px, remove axe labels
             if (withForEachJob < 20) {
```

Comment:
I realize this isn't part of the change on this PR, but can you rename `withForEachJob` to `widthForEachJob`?